controllers/gatekeepersync/gatekeeper_constraint_sync.go

```diff
@@ -345,6 +347,7 @@ func (r *GatekeeperConstraintReconciler) sendComplianceEvent(
 	if len(refreshedPolicy.Status.Details) < templateIndex+1 ||
 		len(refreshedPolicy.Status.Details[templateIndex].History) == 0 ||
 		refreshedPolicy.Status.Details[templateIndex].History[0].Message != fmt.Sprintf("%s; %s", compliance, msg) {
+		//#nosec G401
```

Comment:
Nit: Because it looks prettier
```suggestion
		// #nosec G401
```

Reply:
@JustinKuli mentioned that human comments have spaces after `//` but ones for tools don't. Am I characterizing that right @JustinKuli ?

Reply:
Ah, you're right: https://github.com/securego/gosec#annotating-code
That's lame.

Reply:
I don't remember when I mentioned it, but yeah, I think the general idea is that comments that will be generated or read by machines will not have a space (eg, `go:generate`, `+kubebuilder`, and pragmas). But it's not a strict rule in most cases, and I don't think we're perfectly consistent with the kubebuilder comments anyway